src/Interfaces/MqttTransport.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace ErrorHeroModule\Interfaces;
```

Comment:
if https://github.com/McFizh/libMQTT is the lib you refer, then the service should be an instance of `LibMQTT\ Client`class in documentation config. No need to add this interface.

Reply:
That is the one I use. My idea was to make it more prexible and not dependent to this lib. I could add it to the composer.json file as mandatory. Okay?

Reply:
you can register `mcfish/libmqtt` to composer.json under `require-dev` to allow you write test under `spec` directory, and add under `suggest` to suggest user to require when they want to use mqtt service